src/code/z_actor.c

```diff
@@ -847,7 +847,7 @@ void Actor_Destroy(Actor* actor, GlobalContext* globalCtx) {
     }
 }
 
-void func_8002D7EC(Actor* actor) {
+s16 func_8002D7EC(Actor* actor) {
```

Comment:
The problem is it doesn't seem to return anything, but it still needs to have a return type for matching. I wasn't sure what to set it to.

Reply:
So that would be a fake match? I don't think it's a good idea to edit global functions to make particular actors match because it's (hopefully) not what the original code did, and if it does affect matching it may prevent matching other functions or spread the fake matches.

Reply:
it is very likely what the original code did. we have to do this all the time